script_for_gb_lesson2_task3.js

```diff
@@ -0,0 +1,33 @@
+/*
+3. Дописать игру “Угадай число” (по желанию).
+Сделать игру с двумя игроками и бесконечным числом попыток.
+Сделайте возможность выйти из игры по желанию одного из игроков.
+*/
+alert(`Игра УГАДАЙ ЧИСЛО в режиме двух игроков. Отгадайте число от 1 до 100. (Чтобы выйти из игры, нажмите "Отмена")`);
+let answer = Math.round(Math.random() * 100);
+let playerOneAnswer = prompt("Первый игрок, ваш ответ:");
+let playerTwoAnswer = prompt("Второй игрок, ваш ответ:");
+
+for (;;) {
```

Comment:
Takoi for konechno mojno ispol'zovat', no on vyglyadit dostatochno stranno. Navernoe zdes' luchsche while, no eto ya uje pridirayus'.